cmd/storage-rest-common.go

```diff
@@ -17,13 +17,14 @@
 package cmd
 
 const (
-	storageRESTVersion       = "v17" // RemoveBucket API change
+	storageRESTVersion       = "v18" // Add GetDiskID handler
```

Comment:
can we not update this? @vadmeste since we don't need to break rolling upgrades for this change IMO.